project.clj

```diff
@@ -3,7 +3,7 @@
 (def tk-version "3.1.0")
 (def tk-jetty-version "4.2.1")
 (def tk-metrics-version "1.4.3")
-(def logback-version "1.2.7")
+(def logback-version "1.2.9")
```

Comment:
Is this version available yet? I only see 1.2.8 at https://mvnrepository.com/artifact/ch.qos.logback/logback-classic

Reply:
It must be somewhere, my testing pulled it in correctly...

Reply:
There's no tag for it on GH either... https://github.com/qos-ch/logback/tags

Reply:
But it does show up here: https://repo1.maven.org/maven2/ch/qos/logback/logback-core/

Reply:
http://logback.qos.ch/news.html